src/Features/LanguageServer/Protocol/Handler/Symbols/DocumentSymbolsHandler.cs

```diff
@@ -108,25 +108,26 @@ public async Task<object[]> HandleRequestAsync(RoslynDocumentSymbolParams reques
                 return null;
             }
 
-            var inDocumentInfo = symbolItem.Location.InDocumentInfo.Value;
-            if (inDocumentInfo.spans.Length == 0)
+            var (spans, navigationSpan) = symbolItem.Location.InDocumentInfo.Value;
+            if (spans.Length == 0)
                 return null;
 
-            return new DocumentSymbol
+            return new RoslynDocumentSymbol
             {
                 Name = symbolItem.Name,
                 Detail = item.Text,
                 Kind = ProtocolConversions.GlyphToSymbolKind(item.Glyph),
+                Glyph = (int)item.Glyph,
```

Comment:
Do we have a work item against LSP somewhere to fix the document symbol response here more generally if we also wanted this for actual document symbols too?

Reply:
no.  we do not.  @dibarbet what would be the right way to try to get LSP updated here?  jsut bring up at an LSP meeting?